### PR TITLE
Add stubs for some exercises

### PR DESCRIPTION
**Pull request by TomPradat**
Part of #705 

Can you confirm that I'm on the right tracks ? I'll add some more stubs

### CODE REVIEW

File: exercises/change/change.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+//
+// This is only a SKELETON file for the 'Alphametics' exercise. It's been provided as a
```
Comment (SleeplessByte) [inline exercises/change/change.js]:
```suggestion
// This is only a SKELETON file for the 'Change' exercise. It's been provided as a
```

Comment (TomPradat) [reply exercises/change/change.js]: Copy pasting 🙄 , i'll fix this

File: exercises/zipper/zipper.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+//
+// This is only a SKELETON file for the 'Zipper' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+export default class Zipper {
+  fromTree(tree) {
```
Comment (SleeplessByte) [inline exercises/zipper/zipper.js]:
```suggestion
  static fromTree(tree) {
```

File: exercises/zipper/zipper.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+//
```
Comment (SleeplessByte) [inline exercises/zipper/zipper.js]: Looking at https://github.com/TomPradat/javascript/blob/feature/stubs/exercises/zipper/zipper.spec.js, I don't think the methods in this file are correct. I don't see `left` and `right` for example, and things like `insertBefore` are not methods _under test.

Comment (TomPradat) [reply exercises/zipper/zipper.js]: I get what you mean, but then there is probably a problem with the README.md for this exercise => https://github.com/exercism/javascript/blob/master/exercises/zipper/README.md

Nowhere is it mentioned that there should have `left` or `right` methods.

So the question is should we update the README.md or the example and test files ?

Comment (SleeplessByte) [reply exercises/zipper/zipper.js]: Ahhh. Good question.

The README.md states

> For example given a rose tree (where each node contains a value and a list of child nodes) a zipper might support these operations:

I think we need to bolden the *For example* and we should add a JavaScript Track Hint which methods _are_ added/necessary, OR change the tests to update. I'll look into it <3

Comment (TomPradat) [reply exercises/zipper/zipper.js]: Agreed, thanks 👍

File: exercises/zipper/zipper.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+  insertAfter() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  delete() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  next() {
```
Comment (tejasbubane) [inline exercises/zipper/zipper.js]: `next` function is duplicated. Please remove one.

**Comment by tejasbubane** [comment]
Ahh, I merged https://github.com/exercism/javascript/pull/753 with zipper stub so it created conflict with this PR. Sorry for the confusion. Can you fix the merge conflicts please? @TomPradat

**Pull request merged**